packages/webapp/pages/onboard.tsx

```diff
@@ -0,0 +1,176 @@
+import { useFeaturesContext } from '@dailydotdev/shared/src/contexts/FeaturesContext';
+import { ProgressBar } from '@dailydotdev/shared/src/components/fields/ProgressBar';
+import Logo from '@dailydotdev/shared/src/components/Logo';
+import classNames from 'classnames';
+import ConditionalWrapper from '@dailydotdev/shared/src/components/ConditionalWrapper';
+import { IntroductionOnboardingTitle } from '@dailydotdev/shared/src/components/onboarding/IntroductionOnboarding';
+import AuthOptions from '@dailydotdev/shared/src/components/auth/AuthOptions';
+import { AuthTriggers } from '@dailydotdev/shared/src/lib/auth';
+import { FilterOnboarding } from '@dailydotdev/shared/src/components/onboarding';
+import { cloudinary } from '@dailydotdev/shared/src/lib/image';
+import { Button } from '@dailydotdev/shared/src/components/buttons/Button';
+import { MemberAlready } from '@dailydotdev/shared/src/components/onboarding/MemberAlready';
+import {
+  OnboardingFilteringTitle,
+  OnboardingV2,
+} from '@dailydotdev/shared/src/lib/featureValues';
+import classed from '@dailydotdev/shared/src/lib/classed';
+import { useRouter } from 'next/router';
+import { useAnalyticsContext } from '@dailydotdev/shared/src/contexts/AnalyticsContext';
+import {
+  AnalyticsEvent,
+  TargetType,
+} from '@dailydotdev/shared/src/lib/analytics';
+import { OnboardingStep } from '@dailydotdev/shared/src/components/onboarding/common';
+import { OnboardingMode } from '@dailydotdev/shared/src/graphql/feed';
+import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
+
+const versionToTitle: Record<OnboardingFilteringTitle, string> = {
+  [OnboardingFilteringTitle.Control]: 'Choose topics to follow',
+  [OnboardingFilteringTitle.V1]: 'What topic best describes you?',
+  [OnboardingFilteringTitle.V2]: 'Which topics resonate with you the most?',
+  [OnboardingFilteringTitle.V3]: `Pick the topics you'd love to dive into`,
+  [OnboardingFilteringTitle.V4]: 'Choose the topics you’re passionate about',
+};
+
+const Title = classed('h2', 'font-bold typo-title2');
+
+export function OnboardPage(): ReactElement {
+  const router = useRouter();
+  const { user, isAuthReady } = useAuthContext();
+  const [isFiltering, setIsFiltering] = useState(false);
+  const [isAuthenticating, setIsAuthenticating] = useState(false);
+  const { onShouldUpdateFilters } = useOnboardingContext();
+  const {
+    onboardingFilteringTitle,
+    onboardingMinimumTopics,
+    onboardingV2,
+    isFeaturesLoaded,
+  } = useFeaturesContext();
+  const { trackEvent } = useAnalyticsContext();
+
+  const onClickNext = () => {
+    const screen = isFiltering ? OnboardingStep.Topics : OnboardingStep.Intro;
+
+    trackEvent({
+      event_name: AnalyticsEvent.ClickOnboardingNext,
+      extra: JSON.stringify({ screen_value: screen }),
+    });
+
+    if (!isFiltering) return setIsFiltering(true);
+
+    return setIsAuthenticating(true);
```

Comment:
@idoshamun tracking of clicking the next buttons. There are only two "Next" buttons, in the introduction, and on the topics filtering state.

The tracking of clicking topics is still intact on how it used to work on the previous onboardings.

Reply:
Looks great!